common/src/main/java/io/netty/util/internal/ThreadLocalRandom.java

```diff
@@ -209,6 +225,12 @@ private static long newSeed() {
         }
     }
 
+    private static long mix64(long z) {
```

Comment:
@Scottmitch sure I can ... that said the whole class was at some point ported from there ;)